crawl-ref/source/ouch.cc

```diff
@@ -947,7 +951,8 @@ void ouch(int dam, kill_method_type death_type, mid_t source, const char *aux,
         // don't always trigger in unison when you have both.
         if (you.get_mutation_level(MUT_READ_SAFETY))
         {
-            if (_is_damage_threatening(damage_fraction_of_hp))
+            if (_is_damage_threatening(damage_fraction_of_hp,
+                                        you.get_mutation_level(MUT_READ_SAFETY)))
```

Comment:
If you feel like it, making MUT_DRINK_SAFETY explicitly pass 3 to the function rather than using a default value is slightly nicer. If you feel very ambitious and don't mind if other players get mad at you, certain developers have suggested giving MUT_DRINK_SAFETY the same 3 level mutation treatment and adding it to the general bad mutation pool.

Reply:
to be clear, the way you handle MUT_READ_SAFETY is fine, this is just about where to put the MUT_DRINK_SAFETY parameter since it's not the same.

Reply:
I also was thinking that with these changes the no drink mut could be introduced, since it was also suggested here, I added it as a new regular bad mut.